### ISSUE TITLE
Taking over weekly review of activities

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/tasks#873]
Someone is going to need to take over weekly review of activity submissions

@phet-steele is this something the QA team could handle? Basically you are just making sure it appears to be a legit lesson. In theory we should also be gold starring activities?

@amanda-phet and @arouinfar thoughts?

**Comment by amanda-phet** [comment]
It should be either myself or @arouinfar . 

Can you let us know all of the work that @amyh-phet used to do that needs to be handed off to either me or @arouinfar and let us discuss how to split it up?

**Comment by arouinfar** [comment]
Agreed that @amanda-phet or I should review activities for gold stars. Now that activities must be approved before becoming visible on the site, it might be a good idea for each of us to pick a day to review the unapproved activities so there's less of a delay between activity submission and visibility.

**Comment by amyh-phet** [comment]
@amanda-phet and @arouinfar   I've used this spreadsheet to track activities that were deleted from the website.  Not sure if you want to keep using it.  Trish passed it along to me. 

https://docs.google.com/spreadsheets/d/1QDQYEXNbyoT53vzgQT6BBg-IyM3AFp_GH650OlsK1mU/edit#gid=0

**Comment by amanda-phet** [comment]
That spreadsheet is helpful. Do you have an email template that you use when emailing these authors? Also, what is the response column?

**Comment by amyh-phet** [comment]
Sorry, the response column was incorrectly titled.  I fixed it to say 'Activity Number'.  Trish used to track if she heard back from a user.  Sometimes we get uploads of activities that are blank or incomplete documents.  I typically asked the user to try to upload the document again and gave them a deadline to make changes before just deleting the activity.  I never had a user fix a document and ended up deleting the document anyway.

**Comment by arouinfar** [comment]
Thanks for the clarification @amyh-phet. 

>  I never had a user fix a document and ended up deleting the document anyway.

Have you run into the situation where an activity page had a fleshed out description, etc. but had the incorrect file attached? From what I remember most activities with blank/irrelevant documents don't appear to be genuine activities.  Perhaps we can make a habit of outright deleting the spammy-looking activities, but giving a deadline to those who appear to have a more genuine activity?

**Comment by amyh-phet** [comment]
Here's a link to some canned responses https://docs.google.com/a/colorado.edu/document/d/1fAy5HSTXi7b4ypFChk7iDwXvuQ707cQMwMh6q9Pf2rI/edit?usp=sharing

I made a new folder in Teaching Resources that has the deleted activities spreadsheet and canned responses.

**Comment by amyh-phet** [comment]
@arouinfar   I typically outright delete the spammy activities but send an email to let the author know the activity was deleted.  One big tip off is for when an author aligns the activity to every sim (or a really long list of sims).  If the activity looks legitimate but there is an error with the file, then I'll give the author time to fix and upload a new file.

**Comment by arouinfar** [comment]
Thanks @amyh-phet!

@amanda-phet should we each pick a day to review/approve activities? Maybe M/W or M/Th?

**Comment by arouinfar** [comment]
Spoke with @amanda-phet and I will review activities on Mondays, and she'll do Thursdays. Closing.
